maps/torch/job/security_jobs.dm

```diff
@@ -131,13 +131,13 @@
 	total_positions = 2
 	spawn_positions = 2
 	supervisors = "the entirety of Security"
+	alt_titles = list(
+		"Cadet"
+		)
```

Comment:
Don't see why there's a need for this alt title.

Reply:
right

Reply:
removed